packages/component/src/Activity/private/FeedbackTextArea.tsx

```diff
@@ -0,0 +1,107 @@
+import { hooks } from 'botframework-webchat-api';
+import classNames from 'classnames';
+import React, { forwardRef, Fragment, useCallback, type ChangeEventHandler, type KeyboardEventHandler } from 'react';
+import useStyleSet from '../../hooks/useStyleSet';
+
+const { useUIState } = hooks;
+
+const TextArea = forwardRef<
+  HTMLTextAreaElement,
+  Readonly<{
+    'aria-label'?: string | undefined;
+    className?: string | undefined;
+    'data-testid'?: string | undefined;
+
+    /**
+     * `true`, if the text area should be hidden but stay in the DOM, otherwise, `false`.
+     *
+     * Keeping the element in the DOM while making it invisible to users and PWDs is useful in these scenarios:
+     *
+     * - When the DTMF keypad is going away, we need to send focus to the text area before we unmount DTMF keypad,
+     *   This ensures the flow of focus did not sent to document body
+     */
+    hidden?: boolean | undefined;
+    onChange?: ChangeEventHandler<HTMLTextAreaElement> | undefined;
+    placeholder?: string | undefined;
+    startRows?: number | undefined;
+    value?: string | undefined;
+  }>
+>((props, ref) => {
+  const [uiState] = useUIState();
+  const [{ feedbackTextArea }] = useStyleSet();
+
+  const disabled = uiState === 'disabled';
+
+  const handleKeyDown = useCallback<KeyboardEventHandler<HTMLTextAreaElement>>(event => {
```

Comment:
Remind me there is a bug on macOS + Japanese. We need to look at `oncompositionstart` and `oncompositionend`. And while composition is started, don't do anything here.